### PR TITLE
add ShowOrderBook section

### PR DESCRIPTION
**Pull request by greenhat**

### CODE REVIEW

File: docs/dex-blog-post.md [hunk 1]
```diff
@@ -184,6 +184,33 @@ Doc page:	https://aslesarenko.github.io/ergo-tool/api/org/ergoplatform/appkit/er
...
+```
+
+Here is an example of using `dex:ShowOrderBook`:
+```
+java -jar ../ergotool-3.1.1.jar --conf ergo_tool_config.json dex:ShowOrderBook "56cf33485be550cc32cf607255be8dc8c32522d0539f6f01d44028dc1d190450"
+Loading seller boxes... Ok
+Loading buyer boxes... Ok
+Order book for token 56cf33485be550cc32cf607255be8dc8c32522d0539f6f01d44028dc1d190450:
+Sell orders:
+  Amount   Total(including DEX fee)
```
Comment (aslesarenko) [inline docs/dex-blog-post.md]:
```suggestion
  Token Amount   Erg Amount  (including DEX fee)
```

Please also fix this in the code.

File: docs/dex-blog-post.md [hunk 2]
```diff
@@ -184,6 +184,33 @@ Doc page:	https://aslesarenko.github.io/ergo-tool/api/org/ergoplatform/appkit/er
...
+     100   1005000000
+Buy orders:
+  Amount   Total(including DEX fee)
+     100   1005000000
+```
```
Comment (aslesarenko) [inline docs/dex-blog-post.md:212]: @greenhat The logic of matching require explanation. I.e. which Sell/Buy orders are matched and how DEX fee is computed.

**Review by aslesarenko — CHANGES_REQUESTED** [review]

File: docs/dex-blog-post.md (hunk 1 above)
Comment (greenhat) [reply docs/dex-blog-post.md]: Fixed in https://github.com/ergoplatform/ergo-dex/pull/5

File: docs/dex-blog-post.md (hunk 2 above)
Comment (greenhat) [reply docs/dex-blog-post.md:212]: @aslesarenko What do you mean "matching"? Its sell and buy orders filtered by token id and sorted by total price. I've described it in more details in cmd description in https://github.com/ergoplatform/ergo-dex/pull/5
DEX fee is a parameter in SellOrder and BuyOrder commands and is described there.

**Review by aslesarenko — APPROVED** [review]

**Pull request merged**